src/Entity.php

```diff
@@ -744,6 +779,10 @@ public function post_getFromDB()
     public function post_addItem()
     {
         parent::post_addItem();
+        // If first time, we allow the creation of an entity and switch to recursive mode
+        if (!Session::isMultiEntitiesMode()) {
+            $_SESSION["glpiactive_entity_recursive"] = true;
+        }
```

Comment:
Is this still relevant? I don't think it will trigger since we no longer allow creation.